### PR TITLE
Fix checkout mapping

### PR DESCRIPTION
**Pull request by tobiasbexelius**

### CODE REVIEW

File: src/resolvers/quote-cart/quoteCart.ts [hunk 1]
```diff
@@ -177,8 +180,10 @@ export const QuoteCartSubset: ResolverSubset = {
       const onboardingSession = await upstream.underwriter.getOnboardingSession(
         input.id,
       )
+      console.log(`QUOTECART - ${JSON.stringify(onboardingSession)}`)
```
Comment (robinandeer) [inline src/resolvers/quote-cart/quoteCart.ts:183]: Probably wanna remove these, right?

File: src/api/upstreams/underwriter.ts [hunk 2]
```diff
@@ -41,16 +41,23 @@ export interface OnboardingSessionDto {
   locale: string
   quoteIds: string[]
   campaignId: string | null
-  onboardingSessionSigning: OnboardingSessionSigningDto | null
+  signing: OnboardingSessionSigningDto | null
   tokenCreationUrl: string | null
   signMethods: SignMethod[]
 }
 
 export interface OnboardingSessionSigningDto {
```
Comment (robinandeer) [inline src/api/upstreams/underwriter.ts:49]: Should these be renamed now that "onboarding session" no longer exists?

File: src/resolvers/quote-cart/quoteCart.ts [hunk 3]
```diff
@@ -296,6 +301,35 @@ const transformSignMethods = (dto: SignMethod[]): CheckoutMethod[] => {
   })
 }
 
+const transformSigning = (
+  dto: OnboardingSessionSigningDto | null,
+): Checkout | null => {
+  console.log(`SIGNING: ${dto}`)
```
Comment (robinandeer) [inline src/resolvers/quote-cart/quoteCart.ts:307]: Remove :)

**Review by robinandeer — COMMENTED** [review]
Just come clean up needed -- see comments :)

File: src/api/upstreams/underwriter.ts (hunk 2 above)
Comment (tobiasbexelius) [reply src/api/upstreams/underwriter.ts:49]: Currently everything in UW still follows the old naming so I'll let it be for now

**Review by robinandeer — APPROVED** [review]
Now looks good to me!

**Pull request merged**